packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -166,9 +164,12 @@ class ImageStream extends Diagnosticable {
     if (_completer != null)
       return _completer.removeListener(listener);
     assert(_listeners != null);
-    _listeners.removeWhere((_ImageListenerPair listenerPair) {
-      return listenerPair.listener == listener;
-    });
+    for (int i = 0; i < _listeners.length; ++i) {
+      if (_listeners[i].listener == listener) {
+        _listeners.removeAt(i);
+        continue;
```

Comment:
this should be break?

Reply:
Thanks https://github.com/flutter/flutter/pull/25994